src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -932,6 +933,114 @@ class spell_mage_summon_water_elemental : public SpellScript
     }
 };
 
+#define FingersOfFrostScriptName "spell_mage_fingers_of_frost_proc_aura"
```

Comment:
Isn't it better to use constexpr?

I said it only for my own knowledge xD

Reply:
https://stackoverflow.com/questions/66176642/difference-between-constexpr-and-define
Short: Not here :)